eagleproject/core/static/core/strategist/strategist_validator.js

```diff
@@ -68,6 +70,7 @@ const strategistValidator = (function () {
         masterEl.appendChild(innerEl);
       }
     } else {
+      iVault.parseTransaction({ data: data });
```

Comment:
is there a need for this line outside the try-catch?

Reply:
You are right. Leftover debugging. Sharp eyes.

Reply:
Will fix.

Reply:
Fixed